npm/init.js

```diff
@@ -16,15 +17,30 @@ function validateNpmVersion() {
     }
 }
 
+function download(url) {
+    // We detect proxy by looking at the environment variable
+    var agent = https;
+    if(process.env.https_proxy && process.env.https_proxy.length > 0) {
```

Comment:
Can you please replace
```if(process.env.http...``` with ```if (process.env.http...```
(notice the missing white-space) to remain compatible with the code styling in this file?

Reply:
Done